### PR TITLE
more ensenmble genes ids to cover genes in disease_genes.xlsx

### PR DESCRIPTION
**Pull request by shokrof**
Hey Hiba, Marina, and Thomas,
I  updated the ensemble genes ids
regards,
Moustafa

### CODE REVIEW

File: VariantSurvival_package/R/ensembleTogenes.csv [hunk 1]
```diff
@@ -55,3 +55,1675 @@ PINK1,ENSG00000158828
...
+HRG,ENSG00000113905
+HSD17B10,ENSG00000072506
+HSD17B4,ENSG00000133835
+HSD3B7,ENSG00000099377
+HSPB1,ENSG00000106211
+HSPB8,ENSG00000152137
+HSPG2,ENSG00000142798
+HTRA2,ENSG00000115317
+HTT,ENSG00000197386
+HTTPS://SEARCH.CLINICALGENOME.ORG/KB/GENE-VALIDITY,None
```
Comment (Krannich479) [inline VariantSurvival_package/R/ensembleTogenes.csv:748]: needs to be removed

File: VariantSurvival_package/R/ensembleTogenes.csv [hunk 2]
```diff
@@ -55,3 +55,1675 @@ PINK1,ENSG00000158828
...
+RPE65,ENSG00000116745
+RPGR,ENSG00000156313
+RPL10,ENSG00000147403
+RPS10,ENSG00000124614
+RPS20,ENSG00000008988
+RPS24,ENSG00000138326
+RPS6KA3,ENSG00000177189
+RRAS,ENSG00000126458
+RRAS2,ENSG00000133818
+RS1,None
```
Comment (Krannich479) [inline VariantSurvival_package/R/ensembleTogenes.csv:1351]: needs to be removed

Comment (shokrof) [reply VariantSurvival_package/R/ensembleTogenes.csv]: @Krannich479 fixed

**Comment by Krannich479** [comment]
Thanks Mostafa. I'll test & merge asap

**Comment by Krannich479** [comment]
I close this as it should be solved with PR #26

**Pull request closed without merging**